app/scripts/ui.js

```diff
@@ -335,9 +263,7 @@ async function queryCurrentActiveTab(windowType) {
   return { id, title, origin, protocol, url };
 }
 
-async function initializeUi(activeTab, connectionStream, traceContext) {
-  const backgroundConnection = connectToAccountManager(connectionStream);
```

Comment:
`connectToAccountManager` renamed to `connectSubstreams` and logic moved to top level to get comms connected ASAP.